examples/Fullscreen/Readme.md

```diff
@@ -71,7 +71,7 @@ class FullScreen extends React.Component {
           <h2>Dropped files</h2>
           <ul>
             {
-              files.map(f => <li>{f.name} - {f.size} bytes</li>)
+              files.map((file, index) => <li key={index}>{file.name} - {file.size} bytes</li>)
```

Comment:
I think this happens for some other examples as well, would you mind checking that and update it accordingly?

Reply:
OK. Found another case